src/SwedbankPay.Sdk.Tests/TestHelpers/MockFileProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Primitives;
+using Moq;
+
+namespace SwedbankPay.Sdk.Tests.TestHelpers
+{
+    public class MockFileProvider : IFileProvider
```

Comment:
What is this class used for?

Reply:
Good catch. It seems this file is no longer in use.

Reply:
Actually it was in use i realized now when the build failed, but in Release mode. Switch to Release and check TestHelper class. There is also a missing using statement for Release mode i think.